### PR TITLE
add zigmod support

### PR DESCRIPTION
**Pull request by alichraghi**
after this, u should login to aquila.red and [import scafetch](https://aquila.red/import)

### CODE REVIEW

File: zigmod.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+id: hnjdgzdi5fgqxz3c9374igqivvttkt387jv853l09xdl66jx
+name: scafetch
+license: MIT
+description: a command-line tool to display information about remote git repositories
```
Comment (iddev5) [inline zigmod.yml:4]: Minor nitpick: It should be ``A command line tool``

File: zigmod.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+root_dependencies:
```
Comment (iddev5) [inline zigmod.yml:5]: Dependencies need to be added here, which are zfetch and ay-arg, links to both are mentioned in gyro.zzz

**Review by iddev5 — CHANGES_REQUESTED** [review]
If you are unable to do it right now, I will do it in a while anyways, so its alright 👍

File: zigmod.yml (hunk 2 above)
Comment (alichraghi) [reply zigmod.yml:5]: yes i did't added deps to ask u about dependenices source. [zpm](https://github.com/nektro/zigmod/blob/master/docs/commands/zpm_add.md) or [aquila](https://github.com/nektro/zigmod/blob/master/docs/commands/aq_add.md)?

Comment (iddev5) [reply zigmod.yml:5]: For zfetch, use aquilla and for ay-arg, just regular git dependency

Comment (iddev5) [reply zigmod.yml:5]: After doing some tests, it seems that adding zigmod support is more compilcated.

1) zfetch package on aquila may be outdated, though this is easy to workaround if we use the git package directly
2) it seems zigmod doesnt like if a dep has no build.zig file, which is the case for ay-arg, this needs investigation

Comment (alichraghi) [reply zigmod.yml:5]: yes i tested too. i think it is better to draft this, since the problem exists

**Pull request closed without merging**